### DISCUSSION TITLE
如何将多个配置文件分别设置输出路径

### DISCUSSION THREAD

**Discussion by xiangyi-wang** [xmake-io/xmake#2279] [Q&A] [answered]
```
set_configdir("$(buildir)/config")
add_configfiles("src/xxx.for.in", {pattern = "@([^\n]-)@",filename = "CSMVersion.for",prefixdir='xxx'})
```

试了下这样设置后所有的配置文件都在同一目录下 `prefixdir=`是设置子目录，如果不设置`set_configdir`默认是build路径

**Comment by waruqi** [comment, marked as answer]
目前仅支持一个目录，你可以在 on_config 或者 before_build 里面，走 os.cp 自己复制到任意位置

**Reply by xiangyi-wang** [reply]
好的 谢谢 
另外，我刚试了下Intel fortran编译器，设置module路径不起作用，还是生成到根目录，gfortran是正常的
`set_values("fortran.moduledir", "$(buildir)/mod")`
